src/Autocomplete/Autocomplete.js

```diff
@@ -0,0 +1,218 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { createFilterOptions } from '@material-ui/lab/Autocomplete';
+import useAutocomplete from '@material-ui/lab/useAutocomplete';
```

Comment:
Please use the unstyled component option as we would rather not fight or have an intersection with CSS in JS and TailwindCSS

https://mui.com/customization/unstyled-components/